controller/error_pages.rst

```diff
@@ -193,56 +189,53 @@ for a given status code as HTML or for a given status code and format.
      http://localhost/index.php/_error/{statusCode}
      http://localhost/index.php/_error/{statusCode}.{format}
 
-.. _custom-exception-controller:
-.. _replacing-the-default-exceptioncontroller:
+.. _custom-error-controller:
+.. _replacing-the-default-errorcontroller:
 
-Overriding the Default ExceptionController
+Overriding the Default ErrorController
 ------------------------------------------
```

Comment:
Please change this "tag line" to have exactly the same number of characters as the title